pjsip-apps/src/swig/csharp/pjsua2maui/README.md

```diff
@@ -0,0 +1,126 @@
+# pjsua2maui - pjsip on .net maui 
+
+## STEPS FOR BUILDING:
+
+### Android
+
+- SETUP:
+    - Visual Studio Code & .NET MAUI - [Install steps](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-11.0&tabs=visual-studio-code)
+    - Java 17 
+    - Android NDK r21d
+    - pjsip v2.14.1
```

Comment:
Change this to:
- Android NDKr21d or later
- pjsip v2.14.1 or later

Here I'm using NDK r28 and 2.15.1, and it seems ok